tools/publish-automation/publish.go

```diff
@@ -0,0 +1,269 @@
+	fmt.Println("Hitting ENTER will take you to another terminal to fleet/tools/fleetctl-npm/")
+	fmt.Println("Once there type \"npm publish\". You will need an auth code for that.")
+	fmt.Scanln()
+	fmt.Println("Taking you to the other terminal ... ")
+
+	relativePath := "./tools/fleetctl-npm"
+	err := openTerminalAtPath(relativePath)
+	if err != nil {
+		fmt.Println("Error opening Terminal:", err)
+		return false
+	}
+
+	fmt.Println("Hit ENTER for next step")
+	fmt.Scanln()
+	return true
+}
+
+func openTerminalAtPath(path string) error {
+	var cmd *exec.Cmd
+
+	switch runtime.GOOS {
+	case "darwin":
+		cmd = exec.Command("open", "-a", "Terminal", path)
+	default:
+		return fmt.Errorf("unsupported operating system")
+	}
+
+	err := cmd.Start()
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func checkDocker(version string) bool {
+	fmt.Println("\n===========================================================")
+	fmt.Println("Step 5: Check hub.docker for existing version")
+	fmt.Println("Hitting ENTER will take you to the proper hub.docker page.")
+	fmt.Println("Verify that the version exists")
+	fmt.Scanln()
+	url := "https://hub.docker.com/r/fleetdm/fleet/tags?page=1&name=patch-fleet-" + version
+	fmt.Println("Taking you to the docker hub page: " + url)
+
+	err := openURL(url)
+	if err != nil {
+		fmt.Println("Error opening URL:", err)
+		return false
+	}
+
+	fmt.Println("Does Docker version exist? (Y/y)")
+	var answer string
+	fmt.Scanln(&answer)
+	if answer == "Y" || answer == "y" {
+		return true
+	} else {
+		return false
+	}
+}
+
+func deployDogfood(version string) bool {
+	fmt.Println("\n===========================================================")
+	fmt.Println("Step 6: Deploy " + version + " to Dogfood.")
+	fmt.Println("Hitting ENTER will take you to the proper git action page.")
+	fmt.Println("Once at the git action page do this:")
+	fmt.Println("    1 - Go to #help-infrastructure slack channel and paste this:\"@infrastructure-oncall We have just released " + version + " and are now deploying it to dogfood\"  ")
+	fmt.Println("    2 - press the \"Run workflow\" button on the right side of the screen")
+	fmt.Println("    3 - paste \"fleetdm/fleet:patch-fleet-" + version + "\" as input.")
+	fmt.Println("    4 - press the \"Run workflow\"")
+	fmt.Scanln()
+	url := "https://github.com/fleetdm/fleet/actions/workflows/dogfood-deploy.yml"
+	fmt.Println("Taking you to the docker hub page: " + url)
+
+	err := openURL(url)
+	if err != nil {
+		fmt.Println("Error opening URL:", err)
+		return false
+	}
+
+	fmt.Println("Wait for the script to finish (~10 minutes)?")
+	fmt.Println("Go to dogfood and verify that it works")
+	fmt.Println("SLACK NOTIFICATIONS")
+	fmt.Println("Go to #help-engineering slack channel and paste this:")
+	fmt.Println("     We have just deployed " + version + " to dogfood")
+	fmt.Println("Go to #general slack channel and paste this:")
+	fmt.Println("     :cloud: :rocket: The latest version of Fleet is " + version[1:])
```

Comment:
This is a copy from the slask `osquery/fleet` MOTD, this feels like needles duplication.

Reply:
Looks good to me to keep on both `osquery` and `fleetdm` but I'm open for changes.